packages/jaeger-ui/src/utils/readJsonFile.tsx

```diff
@@ -14,6 +14,24 @@
 
 import JaegerAPI from '../api/jaeger';
 
+function tryParseMultiLineInput(input: any) {
+  const jsonStrings: any[] = input.split('\n').filter((line: string) => line.trim() !== '');
+  const resourceSpansArray: any[] = [];
+
+  jsonStrings.forEach((jsonString: string) => {
+    try {
+      const traceObj = JSON.parse(jsonString.trim());
+      if ('resourceSpans' in traceObj) {
```

Comment:
why is it relevant for this function? All it's supposed to do is parse multi-line text into a single object. It should not be checking that exactly in the JSON strings, this is done later.